### DISCUSSION TITLE
Dark-Mode

### DISCUSSION THREAD

**Discussion by knotenpunkt** [graphql/graphiql#2944] [Q&A / Support]
How can i set and change live the dark-mode programmatically?

**Comment by jonathanawesome** [comment]
You can use the `useTheme` hook via the `@graphiql/react` package. Here's a sandbox [example](https://codesandbox.io/s/graphiql-usetheme-d6hnuy).

**Reply by knotenpunkt** [reply]
thanks @jonathanawesome ,
but im not so familar with react, so i embeeded the graphiQL-console not in a react like website, but more like so:


```javaScript
        // editorTheme
        graphiqlReact = React.createElement(GraphiQL, {
            fetcher: graphQLFetcher,
             query: querydata,

            // editorTheme:null,

            // docExplorerOpen: true
        });


        ReactDOM.render(graphiqlReact, document.getElementById('graphql-stuff'));
```

in that context i want to change the theme at load-time and also if possible later, if it is created and displayed
